airbyte-integrations/connectors/source-mixpanel/source_mixpanel/spec.json

```diff
@@ -4,9 +4,19 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Source Mixpanel Spec",
     "type": "object",
-    "required": ["api_secret"],
     "additionalProperties": true,
     "properties": {
+      "serviceaccount_username": {
+        "title": "Mixpanel Service Account username",
+        "type": "string",
+        "description": "Mixpanel Service Account username"
+      },
+      "serviceaccount_secret": {
+        "title": "Mixpanel Service Account secret",
+        "type": "string",
+        "description": "Mixpanel Service Account secret",
+        "airbyte_secret": true
+      },
```

Comment:
If these are an alternative to `api_secret`, I think you could do something like this: https://github.com/calixa-io/airbyte/blob/master/airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json#L17-L62. The UI displays in a more user-friendly way.

Reply:
Agreed, but that does not look backward compatible, because it adds a ```credentials``` on top of the schema and it will break other Airbyte customers if they don't make this change.

Reply:
Yeah. In that case, I think they can bump a major version change. Let's check that when you open a PR for them!

Reply:
They have to bump a major version change either way, no ?

Reply:
🤔 Yeah, I think so.